ts/Series/AreaRange/AreaRangeSeries.ts

```diff
@@ -268,13 +268,15 @@ class AreaRangeSeries extends AreaSeries {
      */
     public translate(): void {
         const series = this,
-            yAxis = series.yAxis;
+            yAxis = series.yAxis,
+            chart = series.chart;
```

Comment:
You're using `chart` variable in 2 places only.

If you'd like to keep your variable, then it's a good practice to use the smallest possible scope (block scope instead of function scope). (the same with `yAxis` variable - scope could be reduced)
If you don't have many characters in a line (less than 80), then you don't need to declare a `chart` variable at all.

Your call which one you decide :)